src/pages/ReimbursementAccount/NonUSD/Agreements/index.tsx

```diff
@@ -0,0 +1,75 @@
+import type {ComponentType} from 'react';
+import React from 'react';
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useSubStep from '@hooks/useSubStep';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import Confirmation from './substeps/Confirmation';
+
+type AgreementsProps = {
+    /** Handles back button press */
+    onBackButtonPress: () => void;
+
+    /** Handles submit button press */
+    onSubmit: () => void;
+};
+
+const bodyContent: Array<ComponentType<SubStepProps>> = [Confirmation];
+
+function Agreements({onBackButtonPress, onSubmit}: AgreementsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const submit = () => {
```

Comment:
I don't think this declaration is necessary

Reply:
We will do 2 things here once we integrate it with BE.

1. Send it to BE
2. Navigate user to the final screen (this happens now via onSubmit)